src/routes/ProfilePage/ProfilePage.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from 'react';
+import AuthService from '../../services/authService';
+import * as logUtils from '../../components/Utils/Logger';
+import UsersService from '../../services/usersService';
+import { Section } from '../../components/Utils/Utils';
+import './ProfilePage.css'
+import CreateAccountForm from '../../components/CreateAccountForm/CreateAccountForm';
+const log = logUtils.getLogger()
+
+export default class ProfilePage extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      userID: null,
+      user: null,
+      userPreferences: null,
+      edit: false
+    }
+  }
+
+  componentDidMount() {
+    log.debug('Getting user data...');
+    const userID = AuthService.getUserIDFromToken();
+    this.setState({ userID });
+    UsersService.getUser(userID)
+      .then(user => this.setState({ user: user[0] }))
+      .catch(error => console.log(error));
+  }
+
+  handleEditButtonClick = async (e) => {
+    e.preventDefault()
+    this.state.edit
+      ? this.setState({ edit: false })
+      : this.setState({ edit: true })
+  }
+
+  onUpdate = async (e) => {
+    e.preventDefault();
+    window.location.reload()
+  }
+
+  renderProfile() {
+    const { user } = this.state;
+    const firstName = user.FirstName;
+    const lastName = user.LastName;
+    const email = user.EmailAddress;
+    const address = user.Address;
+    const zipCode = user.ZipCode;
+    const lastUpdated = user.LastUpdated.slice(0, 10); // only need date
```

Comment:
Consider using `new Date(user.LastUpdated).toLocaleDateString()` since the date saved in our db is in UTC

Reply:
Mind blown. Thanks, will change!